test/sequence/convert.hpp

```diff
@@ -0,0 +1,51 @@
+/*=============================================================================
+    Copyright (c) 2015 Kohei Takahashi
+
+    Use modification and distribution are subject to the Boost Software
+    License, Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+    http://www.boost.org/LICENSE_1_0.txt).
+==============================================================================*/
+
+#include <string>
+#include <boost/config.hpp>
+#include <boost/core/lightweight_test.hpp>
+#include <boost/fusion/include/convert.hpp>
+#include <boost/fusion/include/at.hpp>
+
+#include <boost/fusion/include/vector.hpp>
+#include <boost/fusion/include/deque.hpp>
+#include <boost/fusion/include/list.hpp>
```

Comment:
Would it be better to include these in the test file that includes this file? There is already a dependency ordering on the include for the `FUSION_SEQUENCE` `#define`.